retrieval-api/src/infra/repos/postgres-retrieval-event-repo.ts

```diff
@@ -0,0 +1,34 @@
+import { RetrievalEvent } from "../../models/retrieval-event";
+import { RetrievalEventRepo } from "../../repos/retrieval-event-repo";
+
+import { Pool, PoolConfig } from "pg";
+
+export class PostgresRetrievalEventRepo implements RetrievalEventRepo {  
+  private pool: Pool;
+  
+  constructor(props?: PostgresRetrivalEventRepoProps) {
+    const poolProps = props?.poolConfig ?? undefined;
+    this.pool = new Pool(poolProps);
+  }
+
+  save(retrievalEvent: RetrievalEvent): Promise<void> {
+    const queryStr = `
```

Comment:
This SQL looks incomplete -- it appears to be capturing only the UUID & CID. But also there are three values inserted with only 2 columns. Is the schema defined in code somewhere or does it live directly in AWS?